hbase-examples/pom.xml

```diff
@@ -33,7 +33,7 @@
     <!--Version of protobuf that hbase uses internally (we shade our pb)
          Must match what is out in hbase-thirdparty include.
     -->
-    <internal.protobuf.version>3.17.3</internal.protobuf.version>
+    <internal.protobuf.version>3.21.7</internal.protobuf.version>
```

Comment:
Maybe there is a way to tie this setting to hbase-thirdparty somehow. Have hbase-thirdparty export a version property in its POM that we can use here.